samples/HubConnectionSample/HubConnectionSample.cpp

```diff
@@ -0,0 +1,81 @@
+#include "stdafx.h"
+
+#include <iostream>
+#include <sstream>
+#include "signalrclient\hub_connection.h"
+
+void send_message(signalr::hub_proxy proxy, const utility::string_t& name, const utility::string_t& message)
+{
+    web::json::value args{};
+    args[0] = web::json::value::string(name);
+    args[1] = web::json::value(message);
+
+    // if you get an internal compiler error uncomment the lambda below or install VS Update 4
+    proxy.invoke<void>(U("send"), args/*, [](const web::json::value&){}*/)
+        .then([](pplx::task<void> invoke_task)  // fire and forget but we need to observe exceptions
+    {
+        try
+        {
+            invoke_task.get();
+        }
+        catch (const std::exception &e)
+        {
+            ucout << U("Error while sending data: ") << e.what();
+        }
+    });
+}
+
+void chat(const utility::string_t& name)
+{
+    signalr::hub_connection connection{U("http://localhost:34281/SignalR")};

```

Comment:
You are right. Will fix. I think I originally created the samples before we decided to append `SignalR` for hubs.